sv_pipeline/exome/load_data.py

```diff
@@ -23,13 +23,16 @@
 QS_COL = 'qs'
 CN_COL = 'cn'
 CALL_COL = 'svtype'
-SAMPLE_COL = 'sample'
+SAMPLE_COL = 'sample_fix'
 NUM_EXON_COL = 'genes_any_overlap_totalexons'
 DEFRAGGED_COL = 'defragmented'
 SC_COL = 'vac'
 SF_COL = 'vaf'
-VAR_NAME_COL = 'name'
+VAR_NAME_COL = 'variant_name'
 GENES_COL = 'genes_any_overlap_ensemble_id'
+PREV_IDENTICAL_COL = 'identical_round1'
+PREV_OVERLAP_COL = 'any_round1'
```

Comment:
Should it be `any_ovl_in_round1`?

Reply:
no because thats not what the column is in the file? its not like we get to pick how the delivered data is formatted